resources/sass/components/blueprints.scss

```diff
@@ -15,6 +15,7 @@
 }
 
 .blueprint-drag-handle {
+    @apply flex-none;
```

Comment:
Do you know what this line is solving? Everything seems to be fine without this, as far as I can tell.

Reply:
This line is solving the problem caused by flex no wrap property.
If the field name go outside the box, it takes all the space and the drag button becomes width 0.
With flex-none you are always sure the button will take his natural width.

![96310477-57db5080-0fff-11eb-8ff5-b1cee20ee4ce](https://user-images.githubusercontent.com/22853052/96603892-c6b4f400-12f4-11eb-8578-449c8a078bfd.png)

Reply:
When I apply all your html changes, it seems fine. The css rule doesn't appear to add anything. Can you confirm the css rule is actually necessary?

Reply:
I know, it's not essential but with this you are sure it will never be affected by flex